polaris.shopify.com/content/tools/stylelint-polaris/rules/colors-declaration-property-value-disallowed-list.md

```diff
@@ -9,7 +9,7 @@ keywords:
 
 ```diff
 // Do
-+ background: var(--p-hint-from-direct-light);
++ background: var(--p-hint-from-direct-lightX);
```

Comment:
So rgba tokens don't exist anymore in the new v11 color palette. Making a temporary change here so we can come back to address this.